Resources/public/js/views/ez-actionbarview.js

```diff
@@ -35,38 +35,38 @@ YUI.add('ez-actionbarview', function (Y) {
                         new Y.eZ.ButtonActionView({
                             actionId: "minimizeActionBar",
                             disabled: false,
-                            label: "Minimize",
+                            label:  Translator.trans('action.minimizeActionBar',{},'eZPlatformUIBundle_js'),
```

Comment:
to follow the same naming convention as others translation files, I would use `editorial` as the context (and then rename eZPlatformUIBundle_js.{en,fr}.xlf to editorial.{en,fr}.xlf)